rest-api/recipe/models/recipe-models.js

```diff
@@ -3,9 +3,52 @@ const db = require('../../../database/dbConfig');
 module.exports = {
   getRecipes,
   getRecipeById,
-  addRecipeTransaction
+  addRecipeTransaction, 
+  editRecipe,
+  editTag,
+  editCategory
 };
 
+//sam
+async function editRecipe(id , recipeUpdate) { 
+  const editRecipe = await db('recipes')
+  .join('users','users.id', 'recipes.user_id')
+  .select(
```

Comment:
Have you tested this? Because doesn't `.update()` need a `.returning()` instead of a `.select()`? :/

Reply:
i haven't actually. Let me look into it